### PR TITLE
Compare arguments from shinydashboard and semantic.dashboard 

### PR DESCRIPTION
**Pull request by maju116**
Added `utils` folder with function for caparison

### CODE REVIEW

File: utils/common_args.R [hunk 1]
```diff
@@ -0,0 +1,27 @@
+library(shinydashboard)
```
Comment (marekrogala) [inline utils/common_args.R]: please add a short comment descripting what this script does

File: .Rbuildignore [hunk 2]
```diff
@@ -2,3 +2,4 @@
 ^\.Rproj\.user$
 .circleci
 examples
+utils
```
Comment (marekrogala) [inline .Rbuildignore]: we discussed that we wanted to call this `development`. `dev_utils` would also be an appropriate name, i think

Comment (maju116) [reply .Rbuildignore]: done

File: utils/common_args.R (hunk 1 above)
Comment (maju116) [reply utils/common_args.R]: done

**Pull request merged**